src/components/ImgEditor/ImgEditor.tsx

```diff
@@ -102,7 +102,10 @@ export default function ImgEditor<T extends FieldValues, K extends Path<T>>(
       <p className="text-xs text-navy-l1 dark:text-navy-l2 mt-2">
         <span>
           Valid types are:{" "}
-          {accept.map((m) => m.split("/")[1].toUpperCase()).join(", ")}.{" "}
+          {accept
+            .map((m) => m.split("/")[1].toUpperCase().replace(/\+xml/gi, ""))
```

Comment:
```suggestion
            .map((m) => m.split("/")[1].toUpperCase())
```
**unrelated to ticket problem**:  convert `SVG+XML`  to  just `SVG`, not sure though if it makes difference or most users know that `SVG+XML` is just `SVG`

Reply:
Agreed! Good catch. I'd guess that most "normal" users would not know what SVG+XML is, or assume it's something special tangentially related to SVG at best, but they would recognize SVG as an option.